Tests/anvil-csharp-tests.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "anvil-csharp-tests",
+    "rootNamespace": "Anvil.CSharp.Tests",
+    "references": [],
```

Comment:
Probably want to reference anvil?

`anvil-csharp-core`

Reply:
Same answer as https://github.com/decline-cookies/anvil-unity-core/pull/84 - the .asmdef automatically references other assemblies as needed by default. If explicit references are preferred, I will add them.